### PR TITLE
Sync with starter template

### PR DESCRIPTION
**Pull request by mickmister**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost-plugin-skype4business/issues/68

### CODE REVIEW

File: build/sync/plan.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
```
Comment (hanzei) [inline build/sync/plan.yml:7]: I'm 1/5 that it's not worth adding `plan.yml` at this point. Once https://mattermost.atlassian.net/browse/MM-28076 is resolved, we can use that.

File: webapp/package.json [hunk 2]
```diff
@@ -7,11 +7,15 @@
   "main": "src/index.js",
```
Comment (hanzei) [inline webapp/package.json:7]: `package.json` in the starter template actually starts with:
```
{
  "private": true,
  "scripts": {
    "build": "webpack --mode=production",
...
```

Should we apply the same here?

**Review by hanzei — APPROVED** [review]
Two suggestions, but nothing blocking

**Review by kosgrz — APPROVED** [review]
It looks great :rocket:

**Review by jfrerich — APPROVED** [review]
LGTM! Lots of files!

**Pull request merged**